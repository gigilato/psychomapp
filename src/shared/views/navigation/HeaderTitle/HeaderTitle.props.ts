export interface IHeaderTitleProps {
  children: string
}
