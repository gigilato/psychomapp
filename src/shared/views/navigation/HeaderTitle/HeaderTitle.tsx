import { Text } from '$atoms'
import { IHeaderTitleProps } from '$navigation/HeaderTitle/HeaderTitle.props'

export const HeaderTitle = ({ children }: IHeaderTitleProps) => {
  return (
    <Text variant="navigation" textTransform="upper-first">
      {children}
    </Text>
  )
}
