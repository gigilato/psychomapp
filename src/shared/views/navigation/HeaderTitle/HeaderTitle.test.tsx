import { render } from '@testing-library/react-native'

import { HeaderTitle } from './HeaderTitle'

describe('HeaderTitle', () => {
  it('renders correctly', () => {
    const { toJSON } = render(<HeaderTitle>Title</HeaderTitle>)
    expect(toJSON()).toMatchSnapshot()
  })
})
