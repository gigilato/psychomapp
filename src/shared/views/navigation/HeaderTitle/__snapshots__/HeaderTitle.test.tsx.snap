// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`HeaderTitle renders correctly 1`] = `
<Text
  style={
    [
      {
        "fontFamily": "SpaceGrotesk-Medium",
      },
      {
        "fontSize": 18,
      },
    ]
  }
>
  Title
</Text>
`;
